packages/adapter/package.json

```diff
@@ -38,8 +38,12 @@
     "type": "git",
     "url": "https://github.com/ioBroker/ioBroker.js-controller/packages/adapter"
   },
-  "scripts": {},
-  "main": "index.js",
+  "scripts": {
+    "build": "tsc -b tsconfig.build.json && npm run copy-files",
+    "copy-files": "cp -r ./src/cert/ ./build/"
```

Comment:
Not sure how to do it the TS way or move such things out of src dir? @AlCalzone

Reply:
If those files are not included in the compilation (imported somewhere), TS won't put them into the output dir.

Reply:
Hm in this case it is a `.crt` file and is read via readFile https://github.com/ioBroker/ioBroker.js-controller/blob/6954bae646f285c2790c4a665905406c02cba343/packages/adapter/lib/adapter/adapter.js#L9589